MapleServer2/PacketHandlers/Game/PremiumClubHandler.cs

```diff
@@ -0,0 +1,73 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using Microsoft.Extensions.Logging;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class PremiumClubHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.VIP_GOODS_SHOP;
```

Comment:
Let's rename the opcode to `PREMIUM_CLUB` since this packet has modes for loading ui, and such.